django/forms/utils.py

```diff
@@ -81,6 +83,10 @@ class ErrorList(UserList, list):
     """
     A collection of errors that knows how to display itself in various formats.
     """
+
+    template_name = 'django/forms/errors/default.html'
+    template_name_ul = 'django/forms/errors/ul.html'
```

Comment:
We seem to have Django templates for these but not Jinja2 templates. Is this on purpose?

Reply:
https://github.com/django/django/pull/12133/files#diff-8df19e1ad45d21d9673c692de1388df72ba80a95579467ef05a3471f2eddeff7

I think we do have these templates for Jinja2?